bpf/common.c

```diff
@@ -103,7 +101,7 @@ static __always_inline void output_ssl_chunk(struct pt_regs *ctx, struct ssl_inf
     if (!add_address_to_chunk(ctx, chunk, id, chunk->fd, info)) {
         // Without an address, we drop the chunk because there is not much to do with it in Go
         //
-        return;
+        //return;
```

Comment:
What the reason of commenting out this `return` statement?

Reply:
Will revert. No longer necessary to comment this out.